plugins/tile/tree.cpp

```diff
@@ -388,16 +410,17 @@ wf::geometry_t view_node_t::calculate_target_geometry()
     /* If view is maximized, we want to use the full available geometry */
     if (view->pending_fullscreen())
     {
-        auto vp = wset->get_current_workspace();
-        int view_vp_x = std::floor(1.0 * geometry.x / size.width);
-        int view_vp_y = std::floor(1.0 * geometry.y / size.height);
-
-        local_geometry = {
-            (view_vp_x - vp.x) * size.width,
-            (view_vp_y - vp.y) * size.height,
-            size.width,
-            size.height,
-        };
+        auto base_geometry = wset->get_attached_output()->get_relative_geometry();
+        local_geometry = adjust_geometry_for_workspace(wset, base_geometry, local_geometry);
+    } else if (this->show_maximized)
+    {
+        auto base_workarea = wset->get_attached_output()->workarea->get_workarea();
+        auto source_geometry = view->get_geometry();
+        local_geometry = adjust_geometry_for_workspace(wset, base_workarea, source_geometry);
+        local_geometry.x     += gaps.left;
```

Comment:
One thing here, we have 3 types of gaps, inner gaps (between tiled views), outer horizontal and vertical gaps for the edges of the output. The `gaps` that you use here, they are calculated based on the normal tiling layout, so some of them can be equal to the inner gaps, others to the outer gaps, depends on how the layout is.

So I think it would make most sense to directly load the outer gaps from wf-config:

```cpp
static wf::option_wrapper_t<int> outer_horiz_gaps{simple-tile/outer_horiz_gap_size};
```

Same for vertical ofc. and use that.

If you are unsure why this is a problem, I would encourage you to try setting the outer gaps to be bigger than the inner gaps, then tile a few views and maximize one of them, you will see the issue :)